src/Qir/Runtime/lib/QSharpCore/README.md

```diff
@@ -0,0 +1,30 @@
+# API Dependency
+
+(See the raw file. Please keep the raw file readable rather than the browser-rendered one)
+
+The listed earlier ones provide the functionality to the listed later ones  
+(the listed later ones include and/or call the listed earlier ones,  
+the listed later ones cannot be compiled into an executable without the listed earlier ones).  
+
+Same-level entities are independent of each other (unless specified otherwise). Entities depend on the levels listed earlier only.  
+
+
+## Level 0. External To This Directory
+**public\CoreTypes.hpp**    (QUBIT, PauliId, RESULT)[, `public\QirTypes.hpp` (QirArray)].  
+**public\QirContext.hpp**   Declares `GlobalContext()`.
+**public\QSharpSimApi_I.hpp**
+                            Defines `IQuantumGateSet`.
+
+## Level 
```

Comment:
Looks like something is missing here?

Reply:
Yep, the Markdown format is missing the automatic enumeration of the headers here :-D... 
Adding the explicit enumeration (but for myself personally would prefer to leave as is :-).